jobs/rabbitmq-server/templates/config-files/advanced.config.erb

```diff
@@ -1,16 +1,4 @@
-<% if p('rabbitmq-server.oauth.enabled') -%>
-[
-  {rabbitmq_auth_backend_oauth2, [
-    {resource_server_id, <<"<%= p('rabbitmq-server.oauth.resource_server_id') %>">>},
-    {key_config, [
-      {default_key, <<"<%= p('rabbitmq-server.oauth.signing_key_id') %>">>},
-      {signing_keys,
-        #{<<"<%= p('rabbitmq-server.oauth.signing_key_id') %>">> => {pem, <<"<%= p('rabbitmq-server.oauth.signing_key') %>">>}}
-      }
-    ]}
-  ]}
-].
```

Comment:
https://github.com/rabbitmq/rabbitmq-server/pull/2833#issue-577666919:
> Symmetric key configuration still requires the old syntax and therefore can be configured using advanced config file.

With the changes in the PR, job property `rabbitmq-server.oauth.signing_key` can't be a symmetric key anymore.
That's okay given that support for symmetric keys will be removed in a future version (see https://rabbitmq.slack.com/archives/C1EDN83PA/p1614346342187000?thread_ts=1614322730.168400&cid=C1EDN83PA).

However, let's adapt the description in https://github.com/pivotal-cf/cf-rabbitmq-release/blob/76a865f3b87aba68d4648dc5ce0c94e9ffaf9b10/jobs/rabbitmq-server/spec#L116 by removing the part "or the symmetric signing key".